nodemon.json

```diff
@@ -0,0 +1,6 @@
+{
+    "env":
+    { 
+        "MONGO_ATLAS_PW": "ange-admin123456"
```

Comment:
remove this file of nodemon because it contains your secrets, try to add secrets in .env